libcolor.sh

```diff
@@ -17,7 +18,9 @@ else
   RED='\033[0;31m'
 fi
 
-function log_debug   { [ "$DEBUG" == "true" ] && echo -e "${NOCOLOR}$1" || true; }
+# shellcheck disable=SC1089
+function log_debug   { [ "$DEBUG" = "true" ] && echo -e "${NOCOLOR}$1" || true; }
```

Comment:
Somehow ash really wants "function" here. I removed function everywhere but had to keep this one. Will probably end up re-adding function where it "visually" should be.